test/test_combined_inner_outer_reduction.cpp

```diff
@@ -165,12 +172,16 @@ TEST_F(NVFuserTest, CombinedSchedulerLayerNormBackward_CUDA) {
     for (auto s : norm_shape) {
       hidden_size *= s;
     }
-    TORCH_CHECK(
-        !fec.getMostRecentKernelRuntime()->isSegmented(),
-        "Fusion shouldn't be segmented! hidden size= ",
-        hidden_size,
-        ", dtype= ",
-        dtype == DataType::Float ? "Float." : "Half.");
+
+    if (dtype == DataType::Half) {
+      TORCH_CHECK(
+          fec.getMostRecentKernelRuntime()->isSegmented() ==
```

Comment:
Segmentation should be fine as long as the input size is not too large

Reply:
This check is removed.